CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No unreleased changes!
+### Added
+- New spec --all task to run all tests (by [@kauredo](https://github.com/kauredo))
+- Tests for output task (by [@kauredo](https://github.com/kauredo))
+
+### Fixed
+- Thor exit_on_failure deprecation warning (by [@kauredo](https://github.com/kauredo))
```

Comment:
These are all fairly straightforward ... but in future it would be great to have separate issues/PRs for each change.